networks/plugins/hashicorp-vault.tf

```diff
@@ -76,9 +90,10 @@ EOF
       content = <<EOF
 #!/usr/bin/dumb-init /bin/sh
 set -e
-echo "cp -a ${local.container_mounted_vault_storage_dir}/. ${local.container_vault_storage_dir}"
-cp -a ${local.container_mounted_vault_storage_dir}/. ${local.container_vault_storage_dir}
-chmod -R 777 ${local.container_vault_storage_dir}
+mkdir ${local.vault_container_certs_dir} ${local.vault_container_vault_storage_dir}
+unzip ${local.vault_container_certs_zip} -d ${local.vault_container_certs_dir}
+unzip ${local.vault_container_vault_storage_zip} -d ${local.vault_container_vault_storage_dir}
+chmod -R 777 ${local.vault_container_vault_storage_dir}
```

Comment:
Maybe using 755 or 700 to make it less controversial

Reply:
777 is required by containerised vault to update the storage files